bin/clean-cache

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import sys, requests, logging, argparse
+from pathlib import Path
+
+# by default, python does not include this path in its search path,
+# so we have to explicitly insert the path to the musher directory
+# in order to import it successfully
+sys.path.insert(0, str(Path(__file__).resolve().parent.parent / "lib"))
+from husky_musher.utils.redcap import LazyObjects
+from husky_musher import configure_logger
+
+base_dir = Path(__file__).resolve().parent.parent
+logging_config_file = base_dir / "logging.yaml"
+configure_logger(logging_config_file)
+
+LOG = logging.getLogger(__name__)
+
+def main(args):
+    data = {
+        'token': LazyObjects.get_project().api_token,
+        'content': 'log',
+        'logtype': 'record_delete',
+        'user': '',
+        'record': '',
+        'beginTime': args.begin_time,
+        'endTime': args.end_time,
+        'format': 'json',
+        'returnFormat': 'json'
+    }
+
+    response = requests.post(LazyObjects.get_project().api_url, data=data)
+    response.raise_for_status()
```

Comment:
You may want to additionally check for 200 status with an error message wrapped as html. (Its not the correct way for the api to respond, but its an issue weve noticed with the redcap api). Check out  the redcap module in id3c for an example of how it handles this.
Additionally - what are your thoughts on incorporating this into that core redcap module?

Reply:
Ah yup, makes sense and I can add that check. I could incorporate it into the core module, but it might make more sense broken out as a function in the redcap utils module within Musher depending on how much we think we want to programmatically access REDCap logs. 

Within the core module though it would be cool to have some functions that get logging like we already have for records. I'd want it to be a more robust function than here though with some additional arguments and checks.